<launch>
<!--  <node pkg="tf" type="static_transform_publisher" name="OdomToBase_footprint" args="0 0 0 0 0 0 1 odom base_link 100" />-->
<!--  <node pkg="tf" type="static_transform_publisher" name="FootprintToLink" args="0 0 0.5 0 0 0 1 base_footprint base_link 100" /> -->
  <node pkg="tf" type="static_transform_publisher" name="LinkToScan" args="0 0 0.5 0 0 0 1 base_link base_scan 100" />
  <node pkg="tf" type="static_transform_publisher" name="LinkToCamera" args="0 0 0.5 0 0 0 1 base_link zed_camera_center 100" />
  <node pkg="gmapping" type="slam_gmapping" name="slam_gmapping" output="screen">
    <remap from="scan" to="scan"/>
    <param name="base_frame" value="base_link"/>
    <param name="odom_frame" value="odom"/>
    <param name="map_update_interval" value="1"/>
    </node>

  <node pkg="urg_node" type="urg_node" name="urg_node">
    <param name="ip_address" value="172.16.0.10"/>
    <param name="frame_id" value="zed_camera_center" />
    <remap from="scan" to="scan" />
  </node>

  <node pkg="robot_pose_publisher" type="robot_pose_publisher" name="local_robot_pose_publisher">
    <remap from="robot_pose" to="robot_pose/local"/>
    <param name="map_frame" value="map"/>
    <param name="base_frame" value="base_link"/>
    <param name="is_stamped" value="true"/>
  </node>
</launch>

